mne/io/proj.py

```diff
@@ -70,6 +70,28 @@ def __deepcopy__(self, memodict):
                 result[k] = v  # kind, active, desc, explained_var immutable
         return result
 
+    def __eq__(self, other):
```

Comment:
I would consider relying on `from mne.utils import object_diff` to the comparison job.

Reply:
Done, that's super handy!